sass/main.scss

```diff
@@ -29,8 +29,6 @@ body {
 }
 
 h1, h2, h3, h4, h5, h6 {
-  display: flex;
-  align-items: center;
   font-weight: bold;
```

Comment:
`display: flex` breaks how inline code blocks are rendered in headings, so I removed it.